game-core/src/main/java/games/strategy/engine/history/History.java

```diff
@@ -59,6 +61,13 @@ public void goToEnd() {
     }
   }
 
+  public @Nullable PlayerId getActivePlayer() {
```

Comment:
Would it be better to return an Optional rather than make it nullable here?

Reply:
Done.